changelog/unreleased/ocs-user-deprovisioning.md

```diff
@@ -0,0 +1,15 @@
+Enhancement: User Deprovisioning for the OCS API
+
+Use the CS3 API and Reva to deprovision users completely.
+
+Two new environment variables introduced:
+```
+OCS_IDM_ADDRESS
+OCS_STORAGE_USERS_DRIVER
+```
+
+`OCS_IDM_ADDRESS` is also an alias for `OCIS_URL`; allows the OCS service to mint jwt tokens for the authenticated user that will be read by the reva authentication middleware.
+
+`STORAGE_USERS_DRIVER` determines how a user is deprovisioned. This kind of behavior is needed since every storage driver deals with deleting differently.
```

Comment:
```suggestion
`OCS_STORAGE_USERS_DRIVER` determines how a user is deprovisioned. This kind of behavior is needed since every storage driver deals with deleting differently.
```

actually `OCS_STORAGE_USERS_DRIVER` and `STORAGE_USERS_DRIVER` are evaluated, maybe clarify